Nodejs/Product/Nodejs/Templates/Files/EmptyTs/EmptyTs.vstemplate

```diff
@@ -8,6 +8,6 @@
     <SortOrder>200</SortOrder>
   </TemplateData>
   <TemplateContent>
-    <ProjectItem SubType="Code" TargetFileName="$fileinputname$.$fileinputextension$" ReplaceParameters="true">EmptyTs.ts</ProjectItem>
+    <ProjectItem SubType="Code" ItemType="TypeScriptCompile" TargetFileName="$fileinputname$.$fileinputextension$" ReplaceParameters="true">EmptyTs.ts</ProjectItem>
```

Comment:
No this is not a bug fix, but it was always specified in our TS templates, so far, I didn't see any difference. So I'll remove this change from both ts and tsx template files.